index.html

```diff
@@ -0,0 +1,874 @@
+              </figure>
+              <h5 class="font-weight-bold"><a href="#">Look at this winter images!</a></h5>
+          </div>
+          <div>
+            <figure class="img-container m-auto">
+              <img src="img/yellowstone.jpg" alt="Yellowstone" class="img-fluid">
+            </figure>
+            <h5 class="font-weight-bold"><a href="#">Look at this winter images!</a></h5>
+          </div>
+          <!-- Small images -->
+          <div>
+            <div class="d-flex posts-small">
+              <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
+              <h6 class="font-weight-bold clearfix"><a href="#">CSS is easier with bootstrap!</a></h6>
+            </div>
+          <hr>
+            <div class="d-flex posts-small">
+              <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
+              <h6 class="font-weight-bold clearfix"><a href="#">CSS is easier with bootstrap!</a></h6>
+            </div>
+          <hr>
+            <div class="d-flex posts-small">
+              <img src="img/bootstrap.png" alt="Image Debate" class="float-left mr-4">
+              <h6 class="font-weight-bold clearfix"><a href="#">CSS is easier with bootstrap!</a></h6>
+            </div>
+          </div>
+        </div>
+      </section>
+    </article>
+    <!-- Subscribe section -->
+    <article class="subscribe bg-white p-2 ">
+        <h6 class="f-category">Subscribe</h6>
+        <section class="d-flex flex-column m-2 subs flex-md-row">
+          <figure class="text-center">
+            <img src="img/subscribe.png" alt="Subscribe" class="img-fluid">
+          </figure>
+          <div class="d-flex flex-column alig-items-md-start subs-text">
+              <h2 class="text-center text-md-left h5 font-weight-bold p-2">In-depth Stories You Don't See Elsewhere About Topics You Want to Read!</h2>
+              <p class="text-center text-md-left p-3">All Access + Weekly Delivery.<br/>Choose A Membership That's Perfect for You!</p>
+              <a href="#" class="text-uppercase text-center text-md-left p-2 anchor-news">Subscribe</a>
+          </div>
+        </section>
+      </article>
+    <!-- Newsletter section  -->
+      <article class="newsletter bg-white p-2">
+        <h6 class="f-category">Newsletter</h6>
+        <figure class="text-center border-muted border m-auto d-md-flex">
+          <img src="img/newsletter.png" alt="Newsletter" class="img-sm-fluid img-md-fluid img-xm-fluid">
+          <figcaption class="text-center border-muted p-3 d-md-flex flex-md-column align-items-start">
+            <h5 class="font-weight-bold">The Starting 5</h5>
+            <p class="text-md-left">Take back your inbox. Sign up for our newsletter and start making sense of the world around you.</p>
+            <a href="#" class="text-uppercase text-center p-2 anchor-news">Sign Up</a>
+          </figcaption>
+        </figure>
+      </article>
```

Comment:
- [ ] The elements in this section are not behaving as expected. Check the image below and see how the newsletter is overflowing the body of your page. If you make use of the suggested classes in the body, you might be able to solve this problem.

- [ ] You are not using the `col-*` class in these elements. You should take advantage of the bootstrap classes to achieve the desired design.

![Screenshot from 2020-03-23 18-31-34](https://user-images.githubusercontent.com/51926937/77375987-b9bb2c80-6d34-11ea-8d3c-a77a8d26d77a.png)

Original:
![Screenshot from 2020-03-23 18-33-29](https://user-images.githubusercontent.com/51926937/77376013-cfc8ed00-6d34-11ea-8953-a92ae97be2e5.png)